lua/lspconfig/async.lua

```diff
@@ -57,6 +57,47 @@ function M.run_command(cmd)
   return stdout and stdout or nil
 end
 
+---@param cmd string[]
+---@param on_done function(string[]?)
+---@param on_error? function(integer, string[]?)
+function M.run_job(cmd, on_done, on_error)
+  local stdout = {}
+  local stderr = {}
+
+  local jobid = vim.fn.jobstart(cmd, {
```

Comment:
Can `vim.system` be used instead?

Reply:
I think `vim.system` might actually be better, the syntax is easier. Maybe I could inline the calls and close this PR.